scripts/setup/constraints.txt

```diff
@@ -49,7 +49,7 @@ colorama==0.4.6
     #   west
 coloredlogs==15.0.1
     # via -r requirements.all.txt
-construct==2.10.54
+construct==2.10.70
```

Comment:
apparently we also have these constraints in:

scripts/setup/requirements.esp32.txt (we can update that one)
third_party/mbed-mcu-boot/repo/boot/espressif/hal/esp-idf/requirements.txt  (unsure about this one ... @dhrishi ?)